src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -28,48 +18,17 @@ import { maxMipLevelCount } from '../../util/texture/base.js';
 
 import { ValidationTest } from './validation_test.js';
 
-class F extends ValidationTest {
-  getDescriptor(
-    options: {
-      width?: number;
-      height?: number;
-      arrayLayerCount?: number;
-      mipLevelCount?: number;
-      sampleCount?: number;
-      format?: GPUTextureFormat;
-    } = {}
-  ): GPUTextureDescriptor {
-    const {
-      width = 32,
-      height = 32,
-      arrayLayerCount = 1,
-      mipLevelCount = 1,
-      sampleCount = 1,
-      format = 'rgba8unorm',
-    } = options;
-    return {
-      size: { width, height, depthOrArrayLayers: arrayLayerCount },
-      mipLevelCount,
-      sampleCount,
-      dimension: '2d',
-      format,
-      usage: GPUTextureUsage.RENDER_ATTACHMENT | GPUTextureUsage.SAMPLED,
-    };
-  }
-}
-
-export const g = makeTestGroup(F);
+export const g = makeTestGroup(ValidationTest);
 
 g.test('zero_size')
   .desc(
     `Test texture creation with zero or nonzero size of
     width, height, depthOrArrayLayers and mipLevelCount for every dimension, and representative formats.
     TODO: add tests for depth/stencil format if depth/stencil format can support mipmaps.`
   )
-  // .cases(poptions('dimension', ['1d', '2d', '3d'] as const))
```

Comment:
Do you think that it is good to make 'dimension' as a parameter in `.cases`, while keep other parameters in `.subcases`? Because 2d or undefined textures are the most important one. 3d textures are not that important, and 1d texture is not important at all (look like we may not support 1d texture in MVP, and GLES 3.2 removed 1d texture). 

If we move `dimension` to `.cases`, then it is easier to see the test results of different dimension types. Then the most important one (2d / undefined, or maybe 3d?) will not mixed with others. While failures in other dimension types are not important. So it is convenient to see the top-priority failures. 

WDYT?

Reply:
I think it makes sense for `dimension` to be a case instead of a subcase.
For example chromium might pass the 2d and 3d cases but need to suppress the 1d case in automated testing.

btw see https://github.com/gpuweb/cts/pull/485/commits/689ff05ec5a284940719291724d97ded03e6c8c2 on how this can be done

Reply:
Great! Done.